WordPress/Classes/ViewRelated/Post/PostEditor+Publish.swift

```diff
@@ -466,50 +651,32 @@ extension PublishingEditor {
 
         mapUIContentToPostAndSave(immediate: true)
 
-        if RemoteFeatureFlag.syncPublishing.enabled() {
-            Task { @MainActor in
-                do {
-                    self.post = try await PostCoordinator.shared._update(post)
-                    if dismissWhenDone {
-                        self.dismissOrPopView()
-                    } else {
-                        self.createRevisionOfPost()
-                    }
-                } catch {
-                    // Do nothing
-                }
-                self.postEditorStateContext.updated(isBeingPublished: false)
-                await SVProgressHUD.dismiss()
-            }
-        } else {
-            consolidateChangesIfPostIsNew()
+        consolidateChangesIfPostIsNew()
 
-            PostCoordinator.shared.save(post,
-                                        defaultFailureNotice: uploadFailureNotice(action: action)) { [weak self] result in
-                guard let self = self else {
-                    return
-                }
-                self.postEditorStateContext.updated(isBeingPublished: false)
-                SVProgressHUD.dismiss()
+        PostCoordinator.shared.save(post, defaultFailureNotice: uploadFailureNotice(action: action)) { [weak self] result in
```

Comment:
Reverting this change as we are now handling the feature flag on a higher level – at the button tap.